packages/next/src/server/use-cache/use-cache-wrapper.ts

```diff
@@ -518,9 +533,10 @@ export function cache(kind: string, id: string, fn: any) {
 
       let stream: ReadableStream
 
+      const disable = true // TODO: Reenable the cache scope
```

Comment:
I found a problem with the cacheScope. It doesn't propagate the revalidate/tags options when there's a hit because we only save the stream - not the full cache entry.